docs/sdk/serverless-signing/zero-to-hero.md

```diff
@@ -0,0 +1,181 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Zero to Hero
+
+## Prerequisites
+
+- Familiarity with JavaScript
+
+## Overview
+In this section we're gonna cover an end-to-end example from minting a PKP through the ContractsSDK, setting its scope & signing a message with it inside a Lit Action.
```

Comment:
consider adjusting copy to:

"The following section provides an end-to-end example of minting a PKP (using the contractsSDK) and signing a message using Lit Actions"